Packs/GoogleThreatIntelligence/Integrations/GoogleThreatIntelligenceASMIssues/GoogleThreatIntelligenceASMIssues.yml

```diff
@@ -0,0 +1,423 @@
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.collection_type
+      description: Type of collection where the issue is categorized.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.organization_uuid
+      description: UUID of the organization linked to the issue.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.pretty_name
+      description: User-friendly name summarizing the issue.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.severity
+      description: Severity level assigned to the issue.
+      type: Number
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.scoped
+      description: Indicates if the issue is scoped to a specific context or asset.
+      type: Boolean
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.confidence
+      description: Confidence score indicating the reliability of the issue detection.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.status
+      description: Current status of the issue.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.category
+      description: Category of the issue based on threat type.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.identifiers.name
+      description: Name of an identifier associated with the issue.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.identifiers.type
+      description: Type of identifier linked to the issue.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.status_new
+      description: Updated status of the issue based on latest assessment.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.status_new_detailed
+      description: Detailed description of the updated issue status.
+      type: String
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.summary.ticket_list
+      description: List of linked tickets associated with the issue.
+      type: list
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.tags
+      description: Tags associated with the issue for classification.
+      type: list
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.cisa_known_exploited
+      description: Indicates whether the issue is part of CISA's Known Exploited Vulnerabilities list.
+      type: Boolean
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.epss_v2_score_lte
+      description: EPSS v2 score indicating likelihood of exploitation (less than or equal).
+      type: Unknown
+    - contextPath: GoogleThreatIntelligenceASMIssues.Issues.epss_v2_percentile_gte
+      description: EPSS v2 percentile indicating exploitation probability (greater than or equal).
+      type: Unknown
```

Comment:
The output type is set to `Unknown`. Please change it to `Number` for consistency.